lib/python/src/bailo/helper/model.py

```diff
@@ -319,6 +336,16 @@ def from_mlflow(self, tracking_uri: str, experiment_id: str):
         if ml_flow:
             client = mlflow.tracking.MlflowClient(tracking_uri=tracking_uri)
             runs = client.search_runs(experiment_id)
+            if len(runs):
+                logger.info(
+                    f"Successfully retrieved MLFlow experiment %s from tracking server. %d were found.",
+                    experiment_id,
+                    len(runs),
+                )
+            else:
+                warnings.warn(
+                    f"MLFlow experiment {experiment_id} does not have any runs and publishing requires at least one valid run. Are you sure the ID is correct?"
+                )
```

Comment:
This is not expected behaviour - MLFlow does not return an error if you request an invalid experiment, instead it will send an empty list. Hence this warning